src/components/FeatureTile/feature-tile.scss

```diff
@@ -23,7 +23,7 @@ a.feature-tile {
   @include breakpoint('md') {
     width: 50%;
     position: absolute;
-    right: 0;
+    right: 1rem;
```

Comment:
Hmm, your fix for #1258 here might conflict with my fix for #1252 here https://github.com/carbon-design-system/carbon-website/pull/1265/files#diff-63c095e1b95ae5b451ab9537ec133b54R35

I added the `ibm-col-bleed` class, but it seems @jeanservaas and @designertyler suggested two different ways to fix the misalignment. 

@jeanservaas @designertyler can you confirm if the image should bleed into the column, or if the card should be moved inside the grid.

Reply:
I think the bleed is correct, based on how the other tiles are displaying, but I could be wrong. 
<img width="1163" alt="screen shot 2019-03-05 at 10 35 26 am" src="https://user-images.githubusercontent.com/2753488/53820945-71b34700-3f32-11e9-9742-14a9480a0401.png">

Reply:
Or with the tile just moved in to line up inside the grid.
<img width="1142" alt="screen shot 2019-03-05 at 10 36 17 am" src="https://user-images.githubusercontent.com/2753488/53821002-8abbf800-3f32-11e9-9758-a5ee8d48b782.png">

Reply:
confirmed with @jeanservaas that your fix is correct 👍